packages/terra-spacer/src/Spacer.jsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { withBps } from 'react-bps';
+import 'terra-base/lib/baseStyles';
+import styles from './Spacer.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Child Nodes.
+   */
+  children: PropTypes.node,
+  /**
+   * Sets top margin. One of 'none', 'small-2', 'small-1', 'small', 'medium', 'large', 'large+1', 'large+2', 'large+3', 'large+4'.
+   */
+  marginTop: PropTypes.oneOf(['none', 'small-2', 'small-1', 'small', 'medium', 'large', 'large+1', 'large+2', 'large+3', 'large+4']),
+  /**
+   * Sets bottom margin. One of 'none', 'small-2', 'small-1', 'small', 'medium', 'large', 'large+1', 'large+2', 'large+3', 'large+4'.
+   */
+  marginBottom: PropTypes.oneOf(['none', 'small-2', 'small-1', 'small', 'medium', 'large', 'large+1', 'large+2', 'large+3', 'large+4']),
```

Comment:
Since all of these are the same it looks like, can you store these in a const array, and then pass that variable on the oneOf?

Reply:
Updated https://github.com/cerner/terra-core/commit/51079e4e1f7a6e3bb384bba084478bb7fcf12335#diff-de1008cdf5bf24f4e07b96189a459cadR8